lib/serial_terminal.pm

```diff
@@ -287,4 +291,84 @@ sub reboot {
     assert_script_run("! test -e '$check_file'");
 }
 
+=head2 select_serial_terminal
+
+ select_serial_terminal($root);
+
+Select most suitable text console. The optional parameter C<root> controls
+whether the console will have root privileges or not. Passing any value that
+evaluates to true will select a root console (default). Passing any value that
+evaluates to false will select unprivileged user console.
+The choice is made by BACKEND and other variables.
+
+Purpose of this wrapper is to avoid if/else conditions when selecting console.
+
+Optional C<root> parameter specifies, whether use root user (C<root>=1, also
+default when parameter not specified) or prefer non-root user if available.
+
+Variables affecting behavior:
+C<VIRTIO_CONSOLE>=0 disables virtio console (use {root,user}-console instead
+of the default {root-,user-}virtio-terminal)
+NOTE: virtio console is enabled by default (C<VIRTIO_CONSOLE>=1).
+For ppc64le it requires to call prepare_serial_console() to before first use
+(used in console/system_prepare and shutdown/cleanup_before_shutdown modules)
+and console=hvc0 in kernel parameters (add it to autoyast profile or update
+grub setup manually with add_grub_cmdline_settings()).
+
+C<SERIAL_CONSOLE>=0 disables serial console (use {root,user}-console instead
+of the default {root-,}sut-serial)
+NOTE: serial console is disabled by default on all but s390x machines
+(C<SERIAL_CONSOLE>=0), because it's not working yet on other machines
+(see poo#55985).
+For s390x it requires console=ttysclp0 in kernel parameters (add it to autoyast
+profile or update grub setup manually with add_grub_cmdline_settings()).
+
+On ikvm|ipmi|spvm|pvm_hmc it's expected, that use_ssh_serial_console() has been called
+(done via activate_console()) therefore SERIALDEV has been set and we can
+use root-ssh console directly.
+=cut
+
+sub select_serial_terminal {
+    my ($root) = @_;
+    $root //= 1;
```

Comment:
Maybe instead of these 2 lines just single line:
```
my $root = shift // 1;
```

Reply:
Thank you!